torch/nn/modules/upsampling.py

```diff
@@ -194,10 +186,6 @@ class UpsamplingBilinear2d(Upsample):
         size (tuple, optional): a tuple of ints `(H_out, W_out)` output sizes
         scale_factor (int, optional): the multiplier for the image height or width
 
-    .. warning::
```

Comment:
I think you missed something here. `UpsamplingNearest2d` is deprecated already before your change. So we should keep this.

Reply:
Oh thanks, will add it back now. :D